deployment/deploy_storybook.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+set -e # Exit with nonzero exit code if anything fails
+PAGES_BRANCH="gh-pages"
+
+if [ "$TRAVIS_PULL_REQUEST" != "false"  ]
```

Comment:
Is the double negation required? Why not `if [ "$TRAVIS_PULL_REQUEST" == "true"  ]`?